browser/src/UI/Actions.ts

```diff
@@ -83,12 +83,15 @@ export interface ISetFont {
 export interface IBufferEnterAction {
     type: "BUFFER_ENTER",
     payload: {
-        id: number,
-        file: string,
-        fileType: string,
-        totalLines: number,
-        hidden: boolean,
-        listed: boolean,
+        currentBuffer: {
```

Comment:
Instead of distinguishing between `currentBuffer` (which is an `IBuffer`) and `existingBuffers`, perhaps it would simplify things to have just a `buffers: IBuffer[]` an an `activeBufferId`?

Reply:
Sounds fair definitely crossed my mind a few times I initially thought there might be a specific need to return the current buffer as a separate item but as you point out its not really needed.